courses/management/utils.py

```diff
@@ -232,3 +232,17 @@ def enroll_in_edx(self, user, course_runs):
         ) as exc:
             self.stdout.write(self.style.WARNING(str(exc)))
         return False
+
+
+def get_course_number(readable_id):
```

Comment:
@rhysyngsun  @pdpinch 
In order to get **course number** from **readable_id**, I have written a this custom function but this looks like a common use-case so what if we move this function to **ol-django -> common -> utils** and do it via opaque_keys, something like this: 
`from opaque_keys.edx.keys import CourseKey

course_key = CourseKey.from_string(enrollment.run.course.readable_id)
course_number = course_key.course
`
What do you suggest?

Reply:
I think it probably justifies the addition of an edx-specific module in `ol-django` as opposed to putting it in `common`

Reply:
Thanks @MAbdurrehman12. I think @rhysyngsun makes a good suggestion.

Reply:
Noted.
Thank you